src/db/enums.rs

```diff
@@ -21,6 +21,18 @@ pub enum ExecutionStatus {
     Failed,
 }
 
+impl From<near_primitives::views::ExecutionStatusView> for ExecutionStatus {
+    fn from(status_view: near_primitives::views::ExecutionStatusView) -> Self {
+        match status_view {
+            // Assume Unknown status as Failed for indexer
```

Comment:
@frol Sorry, I disagree. IMO from indexer perspective it's just a Failed status. If we get outcome with Unknown status it is not something indexer is responsible for, so I think it is the right way to continue working assuming Unknown as Failed instead of crashing. 

If you insist I'd like to hear your rationale so we can decide what to do.

Reply:
Well, IMO, ExecutionStatusView is not perfectly designed. In my understanding, ExecutionOutcome can never have this state (should be refactored on the nearcore side). Thus, I guess, we should just warn about invalid invariant and resolve it as Failed or crash if we want to be pedantic. Failed is also fine for me.